src/redist/minimumMSBuildVersion

```diff
@@ -0,0 +1 @@
+15.3.246
```

Comment:
I think it will get super confusing if the patch version of msbuild can impact resolution. I think we really just want to say which release we need. 

Also, dev builds of msbuild master produce major.minor.0 so this is going to break using those with the resolver.

@AndyGerlicher What do you think?